tools/webpack.assets.json

```diff
@@ -44,55 +69,98 @@
         "./static/js/bundles/common.js"
     ],
     "help": [
+        "./static/js/bundles/common.js",
+        "./static/js/translations.js",
+        "./static/js/portico/header.js",
+        "./static/styles/portico/portico-styles.scss",
```

Comment:
Is there a way for this bundle of 4 lines to become just an include on the portico bundle?  Seems like they get duplicated a lot in this config.

Reply:
Ideally we’d move toward just one JS file per entry point that `import`s everything else it needs. I did this initial pass in the way that involved the least reasoning so I could automate it.